coreneuron/io/output_spikes.cpp

```diff
@@ -226,6 +236,10 @@ void output_spikes_parallel(const char* outpath, const std::string& population_n
 #endif
 
 void output_spikes_serial(const char* outpath) {
+
+    // transfer spikes to NRN
+    all_spikes_return(spikevec_time, spikevec_gid);
+
```

Comment:
`nitpick` : instead of above two callbacks, I think we can add this function call in the beginning of : 

```
268 void output_spikes(const char* outpath, const std::string& population_name) {
```

Reply:
Indeed, thank you